inc/comuni_tipologie.json

```diff
@@ -181,6 +181,11 @@
             "prefix" : "_dci_domanda_frequente_",
             "singular_name" : "Domanda Frequente",
             "plural_name" : "FAQ"
-        }
+        },
+        "page": {
+            "name" : "page",
+            "singular_name" : "Pagina",
+            "plural_name" : "Pagine"
+        },
```

Comment:
Attenzione, con questa virgola alla fine, il JSON non è più valido, e json_decode() lo rifiuta.
Probabilmente è la fonte [dell'errore riscontrato](https://github.com/italia/design-comuni-wordpress-theme/pull/279#issuecomment-1598479418) nel test di integrazione.
[=> COMUNI_TIPOLOGIE non è più un array, e dci_get_group_name() va in errore]

Reply:
si è possibile, grazie! Verifico quanto prima, se hai modo anche tu @abmcr grazie!